site/docs/v1/tech/shared/_user-info-response.adoc

```diff
@@ -0,0 +1,60 @@
+[.api]
+[field]#applicationId# [type]#[UUID]#::
+The unique Id of the Application for which the User has been authenticated.
+
+[field]#birthdate# [type]#[String]# [since]#Available since 1.1.0#::
```

Comment:
Curveball. Since this is included in various places, it'd best to have the `[since]#Available since 1.1.0#` be a parameter. It doesn't make sense to have a parameter 'available since 1.1.0' for an API available since 1.45.

Here's an example of how we handle a similar issue elsewhere: https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/site/docs/v1/tech/events-webhooks/_tenant-scoped-event.adoc